docs/shared/tpl/stats.html

```diff
@@ -12,10 +12,10 @@
         platforms => 100,
 
         # Years since December 18, 1987
-        perl_age => 30,
+        perl_age => 36,
```

Comment:
Note that it's 35 years for a few weeks still.

Reply:
Close enough.